crates/rustc_codegen_spirv/src/codegen_cx/entry.rs

```diff
@@ -36,8 +38,27 @@ impl<'tcx> CodegenCx<'tcx> {
         };
         let fn_hir_id = self.tcx.hir().local_def_id_to_hir_id(local_id);
         let body = self.tcx.hir().body(self.tcx.hir().body_owned_by(fn_hir_id));
+        const EMPTY: ArgAttribute = ArgAttribute::empty();
         for (abi, arg) in fn_abi.args.iter().zip(body.params) {
             if let PassMode::Direct(_) = abi.mode {
```

Comment:
nit: this should probably be turned into a `match`, the reason it's an `if let` is because there's only one case